mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -470,6 +556,38 @@ pub trait UtxoCoinBuilderCommonOps {
         Ok(BlockchainNetwork::Mainnet)
     }
 
+    /// Returns WcChainId for this coin. Parsed from the coin config.
+    fn wallet_connect_chain_id(&self) -> UtxoCoinBuildResult<WcChainId> {
+        let chain_id = self.conf()["chain_id"].as_str().ok_or_else(|| {
```

Comment:
Shouldn't chain_id be part of the protocol similar to EVMs? This will be beneficial if we add UTXO tokens among other things in the future.

Reply:
what protocol?

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/895ac4ad95d268c77546699cd6c7213ac3c38758/CHANGELOG.md#L88-L89